python/py-slack-sdk/Portfile

```
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        slackapi python-slack-sdk 3.1.1 v
+name                py-slack-sdk
+categories-append   irc
+platforms           darwin
+supported_archs     noarch
+license             MIT
+
+python.versions     36 37 38 39
```

```suggestion
python.versions     38 39
```

I'd stick with 38 and 39 for now, and new versions can be added later.

See https://github.com/macports/macports-ports/pull/8620#issuecomment-703742167 for more info.

---

I agree with adding only subports for PY38 and PY39